### DISCUSSION TITLE
Get error when try unit test

### DISCUSSION THREAD

**Discussion by smokeriu** [apache/kyuubi#6200] [Q&A]
I try to run unit test in `kyuubi-spark-sql-engine` module. The test scala file (and package) is created by myself, like: 
```scala
class xxxxSuite extends WithSparkSQLEngine {...}
```

But I got error when try run this test: 

detail: 
```
apache-kyuubi-1.8.1-source/extensions/spark/kyuubi-spark-lineage/src/test/scala/org/apache/kyuubi/plugin/lineage/dispatcher/atlas/AtlasLineageDispatcherSuite.scala:33:8
object KyuubiFunSuite is not a member of package org.apache.kyuubi
import org.apache.kyuubi.KyuubiFunSuite
```
I find `KyuubiFunSuite` in kyuubi-common/test module. 

Other test class use `KyuubiFunSuite` in `kyuubi-spark-lineage` face the same problem.

PS: I've run `mvn package`(skip test) before build unit test. 

IDE: Idea 2023.3

**Comment by beryllw** [comment]
May be you have to run `mvn install`.

**Reply by smokeriu** [reply]
I tried to run `mvn install`(skip test),  but the error remains

**Comment by pan3793** [comment]
> I tried to run `mvn install`(skip test), but the error remains

instead of emphasis "skip test" in comments, why not paste the exact command you run? at least `-Dmaven.test.skip=true` won't work

**Comment by pan3793** [comment]
0. I suppose you are using a UNIX-like OS, e.g. macOS or Linux
1. cd the project's root dir, in your case, should be `apache-kyuubi-1.8.1-source`
2. perform `build/mvn clean install -DskipTests -Pmirror-cdn` (`-Pmirror-cdn` is optional)
3. the IDEA should work then

**Reply by smokeriu** [reply]
I'm using macOs.

I tried your command, the difference is that I added fast profile because `download-module` always fails due to `Read time out`. The full command is:
`build/mvn clean install -DskipTests -Pmirror-cdn,fast`
Strangely, after the execution is complete, the error reported for running the unit test changes, `kyuubi-common` fails directly:
```
Errors occurred while compiling module 'tests of kyuubi-common_2.12'
```
Detail sample:
```
apache-kyuubi-1.8.1-source/kyuubi-common/src/test/scala/org/apache/kyuubi/KerberizedTestHelper.scala:36:36
not found: type KyuubiFunSuite
trait KerberizedTestHelper extends KyuubiFunSuite {
```
```
apache-kyuubi-1.8.1-source/kyuubi-common/src/test/scala/org/apache/kyuubi/KerberizedTestHelper.scala:64:5
not found: value eventually
    eventually(timeout(60.seconds), interval(1.second)) {
```
and more...
